README.md

```diff
@@ -98,6 +98,33 @@ Per default a symlink will by created in /usr/local/bin pointing to your applica
 my-app
 ```
 
+## Creating a Systemd Service
+
+Add a `my-app.service` file to your project containing at least the following
+
+```
+[Unit]
+Description=My awesome app
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/my-app
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Add the `my-app.service` file to an `<ItemGroup>` of your `.csproj` with the following content to make sure that it is installed at the right place.
+
+```
+    <Content Include="my-app.service" CopyToPublishDirectory="PreserveNewest" LinuxFileMode="1755">
+      <LinuxPath>/etc/systemd/system/my-app.service</LinuxPath>
+    </Content>
+```
+
+Make sure to set the Property `<InstallService>true</InstallService>` in your `.csproj` file.
+
+
```

Comment:
## VSCode Build-Menu

Add the following actions to the VSCode build menu: publish as deb or rpm package for x64 / arm64.
Create a folder with the name `.vscode` in the project directory. Create or open a file named `task.json` and insert the following content:

```
{
    "version": "2.0.0",
    "tasks": [
        {
            "label": "dotnet: publish deb (linux-x64)",
            "command": "dotnet",
            "type": "process",
            "args": [
                "publish",
                "${workspaceFolder}/src/your-app/your-app.csproj",
                "/t:CreateDeb",
                "/p:PublishSingleFile=true",
                "/p:PublishTrimmed=true",
                "/p:DebugType=None",
                "/p:RuntimeIdentifier=linux-x64",
                "/p:Configuration=Release"
            ],
            "group": "build",
            "problemMatcher": "$msCompile"
        },
        {
            "label": "dotnet: publish rpm (linux-x64)",
            "command": "dotnet",
            "type": "process",
            "args": [
                "publish",
                "${workspaceFolder}/src/your-app/your-app.csproj",
                "/t:CreateRpm",
                "/p:PublishSingleFile=true",
                "/p:PublishTrimmed=true",
                "/p:DebugType=None",
                "/p:RuntimeIdentifier=linux-x64",
                "/p:Configuration=Release"
            ],
            "group": "build",
            "problemMatcher": "$msCompile"
        },
        {
            "label": "dotnet: publish deb (linux-arm64)",
            "command": "dotnet",
            "type": "process",
            "args": [
                "publish",
                "${workspaceFolder}/src/your-app/your-app.csproj",
                "/t:CreateDeb",
                "/p:PublishSingleFile=true",
                "/p:PublishTrimmed=true",
                "/p:DebugType=None",
                "/p:RuntimeIdentifier=linux-arm64",
                "/p:Configuration=Release"
            ],
            "group": "build",
            "problemMatcher": "$msCompile"
        },
        {
            "label": "dotnet: publish deb (linux-arm64)",
            "command": "dotnet",
            "type": "process",
            "args": [
                "publish",
                "${workspaceFolder}/src/your-app/your-app.csproj",
                "/t:CreateDeb",
                "/p:PublishSingleFile=true",
                "/p:PublishTrimmed=true",
                "/p:DebugType=None",
                "/p:RuntimeIdentifier=linux-arm64",
                "/p:Configuration=Release"
            ],
            "group": "build",
            "problemMatcher": "$msCompile"
        },
        {
            "label": "dotnet: publish rpm (linux-arm64)",
            "command": "dotnet",
            "type": "process",
            "args": [
                "publish",
                "${workspaceFolder}/src/your-app/your-app.csproj",
                "/t:CreateRpm",
                "/p:PublishSingleFile=true",
                "/p:PublishTrimmed=true",
                "/p:DebugType=None",
                "/p:RuntimeIdentifier=linux-arm64",
                "/p:Configuration=Release"
            ],
            "group": "build",
            "problemMatcher": "$msCompile"
        },
    ]
}
```